### PR TITLE
Fixes the permissions spec and related to fixtures to reflect correct action-group names

### PR DESCRIPTION
**Pull request by DarshitChanpura**
### Description

- Re-arranges the fixture load in a test to reflect the correct updates.
- Updates action-group names in fixture files.

### Issues Resolved

- Related to https://github.com/opensearch-project/security-dashboards-plugin/issues/2055

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/plugins/security/permissions_spec.js [hunk 1]
```diff
@@ -73,13 +73,7 @@ if (Cypress.env('SECURITY_ENABLED')) {
     });
 
     it('should create new action group successfully by selecting `Create from blank`', () => {
-      cy.mockPermissionsAction(
```
Comment (DarshitChanpura) [inline cypress/integration/plugins/security/permissions_spec.js:76]: we should not be loading the action group before creating it. The action-group `test-creation` should not be present on initial page load.

**Comment by DarshitChanpura** [comment]
Flaky error;
```
1) Permissions page
       should create new action group successfully by selecting `Create from selection`:
     CypressError: Timed out retrying after 60000ms: `cy.wait()` timed out waiting `60000ms` for the 1st request to the route: `getPermissions`. No request ever occurred.
```

seems to be an issue with cypress, as mentioned here: https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/284

**Review by SuZhou-Joe — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
@DarshitChanpura Seems failed to backport automatically, could you please backport to 2.x manually?

**Comment by DarshitChanpura** [comment]
> @DarshitChanpura Seems failed to backport automatically, could you please backport to 2.x manually?

@SuZhou-Joe Done. Could you please review backports?

**Pull request merged**